keyboards/crkbd/keymaps/shinze/keymap.c

```diff
@@ -0,0 +1,269 @@
+#include QMK_KEYBOARD_H
+#include "bootloader.h"
+#include "keymap_french_osx.h"
+#include "keymap_bepo.h"
+#include "sendstring_bepo.h"
+#ifdef PROTOCOL_LUFA
+  #include "lufa.h"
+  #include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+extern keymap_config_t keymap_config;
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
+
+extern uint8_t is_master;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BEPO  0
+#define _LOWER 3
+#define _RAISE 4
+#define _ADJUST 16
+
+enum custom_keycodes {
+  BEPO = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST
+};
+
+enum macro_keycodes {
+  KC_SAMPLEMACRO,
+};
+
+#define _______ KC_TRNS
```

Comment:
QMK defines this keycode alias automatically.

https://github.com/qmk/qmk_firmware/blob/a120a000d481ed14a87988383be46c5a76604bb6/quantum/quantum_keycodes.h#L29-L31

```suggestion
```